### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by eugenpirogoff**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,23 +2,23 @@
...
 
 ### Access to the Marvel API
 
-In order to actually get data from the Marvel API you will need an [API key](https://developer.marvel.com/signup). 
+In order to actually get data from the Marvel API you will need an [API key](https://developer.marvel.com/signup).
 
 When you have one save the public and private keys in a file named `KeyPair.json` in a `/Resources` directory somewhere. This will ensure that the private key is not committed to version control. For example, to use the Marvelous playground to explore the API save your key pair json in `Resources` directory of the Playground.
 
-The key pair json should have the following format:
+The key pair json in the file should have the following format:
```
Comment (jamiebanford) [inline README.md:21]: Another nitpick: "in the" might be superfluous here? 

> "The key pair json _file_ should have..."

I always try to follow Struck and White's advice that sentences should contain no unnecessary words. Obviously not terribly important, but seeing as I'm, here I thought I'd point it out...

File: README.md [hunk 2]
```diff
@@ -27,4 +27,8 @@ The key pair json should have the following format:
 
 ## Getting help
 
-If you are experiencing problems please [open an issue](https://github.com/diconium/bermob-ios-marvelous/issues) and we will try to address it as soon as we can.
+If you are experiencing problems please [open an issue](https://github.com/diconium/bermob-ios-marvelous/issues) and we will try to address it as soon as we can. Please assign @eugenpirogoff on GitHub to it.
+
+## Prepare for the interview
```
Comment (jamiebanford) [inline README.md:32]: I think gerund is better here: "getting help", "prepar_ing_ for the interview".

File: README.md [hunk 3]
```diff
@@ -27,4 +27,8 @@ The key pair json should have the following format:
...
+
+We assume that you have opened the workspace and gained an overview in the codebase, it does not mean that you know every detail of it, but you know at least what it roughly does. We also assume that you took a look into the [wiki](https://github.com/diconium/bermob-ios-marvelous/projects/1/) and have chosen a issue for us to work on during the interview session. [This here](https://github.com/diconium/bermob-ios-marvelous/issues/10) could be a good starting point.
```
Comment (jamiebanford) [inline README.md:34]: The paragraph updated with some assorted this-native-speaker-is-a-bit-of-a-perfectionist corrections:

> Please clone the repository, open the workspace, and familiarise yourself with the codebase. You don't need to understand every detail, but you at least know what it does. You should also take a look at the [wiki](https://github.com/diconium/bermob-ios-marvelous/projects/1/) and have chosen an issue for us to work on during the pairing session. [This issue](https://github.com/diconium/bermob-ios-marvelous/issues/10) could be a good starting point for example.

**Review by jamiebanford — APPROVED** [review]
👍 

I suggested some tweaks to the text. Nothing too major but I think it would be nice to make them.

**Comment by eugenpirogoff** [comment]
The for the review @jamiebanford . I have updated it.

**Review by jamiebanford — APPROVED** [review]
🚀 

(Sorry for the delay!)

**Pull request merged**